src/orquestra/integrations/qiskit/runner/_ibmq_runner.py

```diff
@@ -41,16 +46,7 @@ def create_ibmq_runner(
     retry_timeout_seconds: int = 24 * 60 * 60,  # default timeout of one day
     discard_extra_measurements=False,
 ):
-    try:
-        IBMQ.enable_account(api_token)
-    except IBMQAccountError as e:
-        if (
-            e.message
-            != "An IBM Quantum Experience account is already in use for the session."
-        ):
-            raise e
-
-    provider = IBMQ.get_provider(hub=hub, group=group, project=project)
+    provider = get_provider(api_token=api_token, hub=hub, group=group, project=project)
```

Comment:
question: To me it looks like `get_provider` only takes two different errors into account. Wherein the previous code ignored all errors except this particular `IBMQAccountError`. How are we ignoring other errors in `get_provider` to preserve this behavior?

Reply:
There are two things here, `IBMQ.enable_account` and `IBMQ.get_provider`.

`IBMQ.enable_account` is no longer a thing in the new API, so that's why it's gone.

I couldn't replicate the `"An IBM Quantum Experience account is already in use for the session."` exception with the new `IBMProvider` path.

However, `IBMQ.get_provider` could also fail and didn't ignore any exceptions which just bubbled up. See `get_qiskit_noise_model` for some of the exceptions that were handled previously.